spinalcordtoolbox/deepseg/models.py

```diff
@@ -100,7 +108,10 @@
          'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel']},
     'seg_exvivo_gm-wm_t2':
         {'description': 'Grey/white matter seg on exvivo human T2w.',
-         'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']}
+         'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']},
+     'seg_gm_sc_7T_t2star':
```

Comment:
```suggestion
     'seg_gm_sc_7t_t2star':
```

i suggest to keep everything in small letter for consistency